README.md

```diff
@@ -90,13 +146,26 @@ The 1Password utility is a light wrapper of the [1Password Connect Python SDK](h
 
 The Slack operator utility makes use of the integration between the Airflow and a Slack app webhook. The purpose of the utility is to add Slack notifications to DAGs using the [callback](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/callbacks.html#callback-types) parameters. Failure, critical failure, and success notifications are implemented.
 
+## Useful Commands
+
+- 🐚 get a shell on a worker, for example
+
+```
+docker exec -it airflow-airflow-worker-1 bash
+```
+
+- ⛔ Stop all containers and execute this to reset your local database.
+  - Do not run in production unless you feel really great about your backups.
+  - This will reset the history of your dag runs and switch states.
+
+```
+docker compose down --volumes --remove-orphans
+```
+
 ## Ideas
-* Make it disable all DAGs on start locally so it fails to safe
-* Create remote worker image example
-  * Use `docker compose` new `profile` support
-* Add slack integration
-* 🤔 Extend webhook to rotate key in 1Password after every use
-  * a true rolling token, 1 use per value
-
-## Example DAGs
-* You can turn on [this field](https://github.com/frankhereford/airflow/blob/main/docker-compose.yaml#L65) to get about 50 example DAGs of various complexity to borrow from
+
+- Make it disable all DAGs on start locally so it fails to safe
```

Comment:
This is a comment from my notes-to-self thoughts made during the initial development that has made it here into this README. It can certainly be improved in how it shares this idea:

I had imagined a mechanism that detects a start-up of the airflow stack with the environment context set to `development`. When this happens, this mechanism would run though all DAGs which are installed on the local stack and turn them off. The idea being that this eliminates the risk of a DAG which updates an outside data source or target from running before the developer realizes it and intervenes. Certainly, we'd never knowingly setup a DAG that does this, but I could imagine something being mid-development that might do something on execution that a developer didn't intend. 

I'd prioritize this enhancement at "ultra-low" and also be totally fine striking it from the ideas list.